functions/package.json

```diff
@@ -18,17 +18,20 @@
     "@discordjs/rest": "^0.5.0",
     "@google-cloud/bigquery": "^6.2.0",
     "@google-cloud/pubsub": "^2.19.4",
+    "@mailchimp/mailchimp_marketing": "^3.0.80",
     "discord-api-types": "^0.32.1",
     "dotenv": "^16.0.1",
     "ethers": "^5.6.8",
     "firebase-admin": "^9.8.0",
     "firebase-functions": "^3.24.1",
+    "jest": "^29.5.0",
```

Comment:
test dependencies should be in "devDependencies" - please change and make another PR

Reply:
changed and pushed some changes to main, mainly hotfixes that were already in my local branch and were not pushed. Fixed dev dependencies too.